.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,24 @@
+# Release Description
+<!--- Summarize what is being released.  -->
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+- [ ] Version number has been updated
+- [ ] Description file updated with New Developers (if applicable)
+- [ ] NEWS.md has been updated and issues linked where appropriate
+- [ ] Ensure all unit tests are passing and Test Coverage is Green on Readme
+- [ ] Ensure CMD Checks are passing on Readme
+- [ ] GitHub actions on this PR are all passing
+- [ ] Draft GitHub release created using automatic template and updated with additional details. Remember to click "release" after PR is merged.
+- [ ] Website is deployed and visual checks for integrity completed
+- [ ] Prepare and Post announcements for Linked-In and Slack Channels for Release
+
+# Clean up Checklist
+<!--- Fill out the following Clean up checklist -->
+- [ ] Delete branches that are stale or abandoned 
+- [ ] Delete branches from merged Pull Requests that made it into `devel`
+- [ ] Triage and close issues that were not closed properly via PRs that made it into release 
+- [ ] Close Pull Requests that are stale 
+- [ ] Set up 0.8.0 News/Changelog Skeleton
```

Comment:
@bms63 you'd want to make this one more generic rather than saying 0.8.0 (e.g. say "next release number")

Reply:
Good call!  I will make it more generic